src/platform/site-wide/announcements/tests/components/SingleSignOnInfoModal.unit.spec.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+import sinon from 'sinon';
+
+import localStorage from 'platform/utilities/storage/localStorage';
+import SingleSignOnInfoModal from '../../components/SingleSignOnInfoModal';
+
+describe('Announcements <SingleSignOnInfoModal>', () => {
+  it('does not render for users that are logged in but without SSO', () => {
+    const tree = mount(<SingleSignOnInfoModal isLoggedIn />);
+
+    expect(tree.html()).to.equal(null);
+
+    tree.unmount();
+  });
+  afterEach(() => {
+    localStorage.clear();
+  });
```

Comment:
Not sure if you put this after the previous case because you thought it would only apply to subsequent blocks, but this will apply to anything in this `describe` block, including the block above it.

Reply:
accidentally had it there instead of the top, will move for clarity